app/models/activity.rb

```diff
@@ -0,0 +1,7 @@
+class Activity < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_and_belongs_to_many :groups
+
+  validates :name, presence: true, length: { maximum: 50 }
+  validates :amount, numericality: { greater_than_or_equal_to: 0 }
+end
```

Comment:
**Optional**
Nice job here. However according to the requirement in the `transaction page`: For a given category, the list of transactions is presented ordered by the most recent. Currently, it's the reverse. See screen-shot below.

![image](https://user-images.githubusercontent.com/51296741/148590036-1a363fa0-b478-45f0-9243-45eadb499826.png)

I will suggest you fix this before merging the PR.

Reply:
Thanks for reminding me about that. I will do it. 👍